README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
OK - algorithm tuning. GridSearchCV will take a 'grid' of parameters (think matrix master), where a sample of values is specified for each parameter being tested. The CV will then test and score every combination, and make the best combination available in the "best_estimator". 

Didn't do any work with http://scikit-learn.org/stable/modules/grid_search.html#randomized-parameter-optimization, but I think I get the gist.
Rather than explicitly defining a finite grid, provide ranges or discrete sets and a number of iterations to run through - and it will randomly test different combinations.